internal/common/common.go

```diff
@@ -18,6 +18,13 @@ const HostCACertPath = "/etc/assisted-service/service-ca-cert.crt"
 
 const consoleUrlPrefix = "https://console-openshift-console.apps"
 
+const MirrorRegistriesCertificateDir = "/etc/pki/ca-trust/extracted/pem"
```

Comment:
This one is only used in L23. Does it have to be a const? maybe there's a use case I haven't considered.

Reply:
since i need to use PATH and FILE, i would prefer if PATH would have been constructed also from const. In addition, in case we want to access that directory in the future, we will already have a const for it

Reply:
> since i need to use PATH and FILE, i would prefer if PATH would have been constructed also from const. In addition, in case we want to access that directory in the future, we will already have a const for it

This sounds like an early optimization we don't need. and it makes reading the series of const variables more confusing. We can always split it and create the const in the future if needed. Not a big deal, I guess.

Reply:
fixed